src/ServiceControl.Transports.RabbitMQ/RabbitMQQuery.cs

```diff
@@ -154,7 +157,7 @@ public override async IAsyncEnumerable<QueueThroughput> GetThroughputPerDay(IBro
             throw new Exception("The RabbitMQ broker is configured with 'management.disable_stats = true' or 'management_agent.disable_metrics_collector = true' and as a result queue statistics cannot be collected using this tool. Consider changing the configuration of the RabbitMQ broker.");
         }
 
-        var rabbitVersion = obj["rabbitmq_version"] ?? obj["product_version"];
+        var rabbitVersion = obj["rabbitmq_version"] ?? obj["product_version"] ?? obj["lavinmq_version"];
```

Comment:
Will this report the LavinMQ version number as the RabbitMQ version in the broker metadata?  

I'm not sure that's what we want to do.  Shouldn't we identify it's actually LavinMQ and not RabbitMQ somehow?

Reply:
At this stage - this is just a spike. It is all "wrong" as the transport is rabbitmq and all docs etc, including SP display will say rabbitmq